src/components/navigation.js

```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from "react";
+import { GithubIcon } from "./github-icon";
+import { InstagramIcon } from "./instagram-icon";
+import { NavigationLink } from "./navigation-link";
+import { PatreonIcon } from "./patreon-icon";
+import { YoutubeIcon } from "./youtube-icon";
+
+// FIXME
+export function Navigation() {
+  const [open, setOpen] = useState(false);
+
+  // Close the menu when clicking on a link.
+  useEffect(() => {
+    const onHashChange = () => {
+      setOpen(false);
+    };
+
+    window.addEventListener("hashchange", onHashChange);
```

Comment:
Do we also want to close the nav menu if the user clicks outside of it (maybe that for later)?

Reply:
That's a great idea, yes! I'm actually thinking about reimplementing this component with something from Radix UI, where these kind of things are automatically handled for us. Another thing, for instance, is closing the menu when the user pressed Escape.